ale_linters/elixir/credo.vim

```diff
@@ -0,0 +1,54 @@
+" Author: hauleth - https://github.com/haulethe
+
+if exists('g:loaded_ale_linters_elixir_credo')
+  finish
+endif
+
+let g:loaded_ale_linters_elixir_credo = 1
+
+function! ale_linters#elixir#credo#Handle(buffer, lines)
+  " Matches patterns line the following:
+  "
+  " file.go:27: missing argument for Printf("%s"): format reads arg 2, have only 1 args
+  " file.go:53:10: if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)
+  " file.go:5:2: expected declaration, found 'STRING' "log"
```

Comment:
Are these comments wrong? I look like the examples are for the Go linter instead.